src/mjolnir/ingest_transit.cc

```diff
@@ -425,13 +436,14 @@ bool write_stop_pairs(Transit& tile,
             origin_is_generated ? origin_stopId
                                 : origin_stopId + "_" + to_string(NodeType::kMultiUseTransitPlatform);
         stop_pair->set_origin_onestop_id(origin_onestop_id);
+
         bool dest_is_generated = tile_info.tile_station_children.find({dest_stopId, currFeedPath}) !=
                                  tile_info.tile_station_children.end();
         auto dest_onestop_id =
             dest_is_generated ? dest_stopId
                               : dest_stopId + "_" + to_string(NodeType::kMultiUseTransitPlatform);
         stop_pair->set_destination_onestop_id(dest_onestop_id);
-        stop_pair->set_origin_graphid(kInvalidGraphId);
```

Comment:
the first time i started debugging this problem i thought i could help myself by initializing these to invalid but what i forgot was that this is protobuf 2 so later on it easily just calls `has_origin_graphid` rather than checking for our sentinel value. same for destination below